src/main/views/hearing-list.njk

```diff
@@ -8,8 +8,11 @@
 
 {% block beforeContent %}
   {{ govukBackLink({
-    text: "Back",
-    href: referringPage
+    text: text,
```

Comment:
Should we use goBack from common-components for this as well? Or we should replace goBack with back link from `govuk` library?

Reply:
i think just in here as this page can be arrived from 2 different routes, it may be more appropiate to go back to which ever side they came from by getting history rather than manually routing them backwards like we have done on the others

Reply:
becuase this page can be landed on from the search route or the alphabet route, so adding a href of  '/search-result' wouldnt fit if they came from alphabet side